### PR TITLE
WV host tool chain environment variable added for ninja build

### PR DESCRIPTION
**Pull request by Haseenamol**
Tested with/without enabling ninja

### CODE REVIEW

File: package/opencdm/opencdm.mk [hunk 1]
```diff
@@ -8,8 +8,8 @@ OPENCDM_SITE_METHOD = git
 OPENCDM_SITE = https://github.com/WebPlatformForEmbedded/WPEOpenCDM.git
 
 OPENCDM_INSTALL_STAGING = YES
-OPENCDMI_LICENSE = Apache-2.0
-OPENCDMI_LICENSE_FILES = LICENSE
+OPENCDM_LICENSE = Apache-2.0
```
Comment (charlie-ht) [inline package/opencdm/opencdm.mk]: Perhaps these unrelated changes should be in a separate patch.

File: package/widevine/widevine.mk [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # widevine
 #
 ################################################################################
-WIDEVINE_VERSION = a4998dc60e358c436ef56577f94ac2ae793f09ce
+WIDEVINE_VERSION = 315655c6ebf58a696b6dde78b14db7aad527e32f
```
Comment (charlie-ht) [inline package/widevine/widevine.mk]: What's this about? Seem unrelated to the patch

File: package/widevine/widevine.mk [hunk 3]
```diff
@@ -30,22 +32,14 @@ define WIDEVINE_CONFIGURE_CMDS
...
 define WIDEVINE_INSTALL_TARGET_CMDS
         cp $(@D)/out/arm/Debug/widevine_ce_cdm_unittest $(TARGET_DIR)/usr/bin
-        cp $(@D)/out/arm/Debug/lib.target/lib*.so $(TARGET_DIR)/usr/lib/
+        cp $(@D)/out/arm/Debug/lib*/lib*.so $(TARGET_DIR)/usr/lib/
 endef
 
 define WIDEVINE_INSTALL_STAGING_CMDS
         cp $(@D)/out/arm/Debug/widevine_ce_cdm_unittest $(STAGING_DIR)/usr/bin
-        cp $(@D)/out/arm/Debug/lib.target/lib*.so $(STAGING_DIR)/usr/lib/
+        cp $(@D)/out/arm/Debug/lib*/lib*.so $(STAGING_DIR)/usr/lib/
```
Comment (charlie-ht) [inline package/widevine/widevine.mk]: Why change to copying the static archives? I thought we only needed the shared lib in the target dir

File: package/opencdm/opencdm.mk (hunk 1 above)
Comment (Haseenamol) [reply package/opencdm/opencdm.mk]: Ok will remove it from this

File: package/widevine/widevine.mk (hunk 2 above)
Comment (Haseenamol) [reply package/widevine/widevine.mk]: Bumbed the widevine to the latest to get env changes

File: package/widevine/widevine.mk (hunk 3 above)
Comment (Haseenamol) [reply package/widevine/widevine.mk]: Not copying static achieve, only *.so is copying. 
using make build system, libwidevine_ce_cdm_shared.so is getting generated in out/arm/Debug/lib.target/, but for ninja it is in out/arm/Debug/lib/ so that this changed to lib*/lib*.so

Comment (charlie-ht) [reply package/widevine/widevine.mk]: Thanks for comments.
